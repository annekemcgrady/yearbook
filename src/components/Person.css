.person-card {
  height: 400px;
  width: 250px;
  background-color: white;
  background: rgba(245, 241, 241, 0.9);
  border-radius: 20px;
  margin: 10px;
  padding: 15px 20px;
  box-shadow: 10px 10px 5px 0px rgba(0, 0, 0, 0.527);
}
  
  img {
    height: 250px;
    width: 200px;
    border-radius: 20px;
  }

  .name {
    color: red;
  }

  .superlative {
    color: #6495EC;
    font-weight: 600;
  }