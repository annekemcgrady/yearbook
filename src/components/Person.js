import React from 'react';
import './Person.css';

const Person = props => {
  console.log(props.id)
  return (
    <div key={props.id} className="person-card">
      <img src={props.photo} alt=''></img>
      <h3 className="name">{props.name}</h3>
      <p className="quote">{props.quote}</p>
      <p className="superlative">{props.superlative}</p>
    </div>
  )
}

export default Person;
