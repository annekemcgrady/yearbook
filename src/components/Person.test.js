import React from 'react';
import ReactDOM from 'react-dom';
import { shallow, mount } from 'enzyme'
import Person from './Person.js';

describe('Person', () => {
  it('should match the snapshot when person data is passed in correctly', () => { 
    const wrapper = shallow( 
    <Person 
      id={1}
      name="Jane"
      photo="https://placekitten.com/200/300"
      quote="too cool for school"
      superlative="most likely to be awesome"
    />);

    expect(wrapper).toMatchSnapshot()
  })
})