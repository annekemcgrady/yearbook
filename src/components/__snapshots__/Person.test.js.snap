// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Person should match the snapshot when person data is passed in correctly 1`] = `
<div
  className="person-card"
  key="1"
>
  <img
    alt="portrait"
    src="https://placekitten.com/200/300"
  />
  <h3
    className="name"
  >
    Jane
  </h3>
  <p
    className="quote"
  >
    too cool for school
  </p>
  <p
    className="superlative"
  >
    most likely to be awesome
  </p>
</div>
`;
